packages/rum/wdio.conf.js

```diff
@@ -25,37 +25,24 @@
 
 const { join } = require('path')
 const glob = require('glob')
-const { getSauceConnectOptions } = require('../../dev-utils/test-config')
+const {
+  getSauceConnectOptions,
+  getBrowserList
+} = require('../../dev-utils/test-config')
 const { isChrome } = require('../../dev-utils/webdriver')
 
 const { tunnelIdentifier, username, accessKey } = getSauceConnectOptions()
-const browserList = [
-  {
-    browserName: 'chrome',
-    version: '62'
-  },
-  {
-    browserName: 'firefox',
-    version: '57'
-  },
-  {
-    browserName: 'microsoftedge',
-    platform: 'Windows 10',
-    version: '17'
-  },
-  {
-    browserName: 'safari',
-    platform: 'OS X 10.11',
-    version: '9.0'
-  },
-  {
-    appiumVersion: '1.9.1',
-    deviceName: 'android emulator',
-    browserName: 'browser',
-    platformVersion: '5.0',
-    platformName: 'android'
-  }
-].map(capability => ({ tunnelIdentifier, ...capability }))
+
+/**
+ * Skip the ios platform on E2E tests because of script
+ * timeout issue in Appium
```

Comment:
does this means we'll not be running E2E tests on iOS?

Reply:
Yeah, hoping for saucelabs to release a new version of appoium dist. They currently only have 1.9 whereas new appium is already on 1.12. We never ran before, But other tests like integration and unit is covered.

Reply:
OK